vuu-ui/packages/vuu-filters/src/filter-toolbar/useFilterToolbar.tsx

```diff
@@ -52,21 +49,20 @@ const filterToControl = (
         key={column}
         labelPlacement="top"
       >
-        <FilterDropdown
+        { values.map((value, idx) => <FilterDropdown
```

Comment:
This looks a bit odd. Instead os a single multi select dropdown that will show the filter values, we have multiple dropdowns each showing one value, was this intended ?

Reply:
Agree that this is incorrect, I think I misunderstood what was happening with this component - rolling back the change